.ci/jobs/cloudbeat-mbp.yml

```diff
@@ -0,0 +1,55 @@
+---
+- job:
+    name: cloudbeat/cloudbeat-mbp
+    display-name: CloudBeat-MBP
+    description: Build & package cloudbeat
+    project-type: multibranch
+    concurrent: true
+    script-path: Jenkinsfile
+    scm:
+    - github:
+        branch-discovery: no-pr
+        discover-pr-forks-strategy: merge-current
+        discover-pr-forks-trust: permission
+        discover-pr-origin: merge-current
+        discover-tags: true
+        head-filter-regex: '(main|6\.8|7\.1[6789]|8\.\d+|PR-.*|v[0-9].*)'
+        notification-context: 'internal-ci'
+        repo: cloudbeat
+        repo-owner: elastic
+        credentials-id: 2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken
+        ssh-checkout:
+          credentials: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+        build-strategies:
+        - tags:
+            ignore-tags-older-than: -1
+            ignore-tags-newer-than: -1
+        - named-branches:
+            - exact-name:
+                name: 'main'
+                case-sensitive: true
+            - regex-name:
+                regex: '7\.1[6789]'
+                case-sensitive: true
```

Comment:
case-sensitive means anything in this context? these are numbers

Reply:
It's a regex, not sure if it will match a suffix like "8.x.x-alpha". I prefer to keep it restrictive in any case and match branches exactly.